applications/asset_tracker_v2/src/modules/debug_module.c

```diff
@@ -257,38 +246,6 @@ static void send_memfault_data(void)
 	}
 }
 
-static void add_location_metrics(uint8_t satellites, uint32_t search_time,
-				 enum location_module_event_type event)
-{
-	int err;
-
-	switch (event) {
-	case LOCATION_MODULE_EVT_GNSS_DATA_READY:
-		err = MEMFAULT_METRIC_SET_UNSIGNED(gnss_time_to_fix_ms, search_time);
-		if (err) {
-			LOG_ERR("Failed updating gnss_time_to_fix_ms metric, error: %d", err);
-		}
-		break;
-	case LOCATION_MODULE_EVT_TIMEOUT:
-		err = MEMFAULT_METRIC_SET_UNSIGNED(location_timeout_search_time_ms, search_time);
-		if (err) {
-			LOG_ERR("Failed updating location_timeout_search_time_ms metric, error: %d",
-				err);
-		}
-		break;
-	default:
-		LOG_ERR("Unknown location module event.");
-		return;
-	}
-
-	err = MEMFAULT_METRIC_SET_UNSIGNED(gnss_satellites_tracked_count, satellites);
-	if (err) {
-		LOG_ERR("Failed updating gnss_satellites_tracked_count metric, error: %d", err);
-	}
-
-	memfault_metrics_heartbeat_debug_trigger();
```

Comment:
I didn't realize the debug trigger was called here, this should be really nice to clean up some of the heartbeats that are generated

Reply:
Agreed! This has been on our list to remove.